src/redux-store/reducers/auths.js

```diff
@@ -16,10 +16,11 @@ export const authReducer = (state = initialState, action) => {
         loading: true
       };
     case types.LOGIN_SUCCESS:
+      localStorage.setItem("token", action.payload);
```

Comment:
remove this, we might not need it, setting token to local storage is already being handled by a middleware